bigflow/cli.py

```diff
@@ -318,15 +320,41 @@ def _add_build_dags_parser_arguments(parser):
                         type=bf_commons.valid_datetime)
 
 
+def _add_build_image_parser_arguments(parser: argparse.ArgumentParser):
+    parser.add_argument(
+        '--export-image-tar', dest='export_image_tar', action='store_true',
+        help="Export built docker image into .tar file",
+    )
+    parser.add_argument(
+        '--no-export-image-tar', dest='export_image_tar', action='store_false',
+        help="Don't epxort built docker image into .tar file (keep image in local docker registry)",
+    )
+    parser.set_defaults(export_image_tar=None)
+
+    parser.add_argument(
+        '--cache-from-image', dest='cache_from_image',
+        help="Docker images to consider as cache sources",
+    )
+    parser.add_argument(
+        '--cache-from-version', dest='cache_from_version',
+        help="Use previous version of the project as cache source",
```

Comment:
If I understand correctly it doesn't have to be the previous version? It can be even a very old version from two years?

Reply:
Yes, It may be any version.

Reply:
So maybe I Would change `help` section of this method to `"Use older version of the project as cache source",`. For me "previous version" suggests only one last version